src/store/getters/state-getters.ts

```diff
@@ -100,11 +100,8 @@ export const isGovAccountUser = (state: StateIF): boolean => {
 
 /** Whether the user is SBC Staff (which is not the same as Staff). */
 export const isSbcStaff = (state: StateIF): boolean => {
-  if (isGovAccountUser(state)) {
-    const orgInfo = getOrgInformation(state)
-    return (orgInfo?.branchName?.includes('Service BC') || false)
-  }
-  return false
+  const orgInfo = getOrgInformation(state)
+  return (orgInfo?.orgType?.includes('SBC_STAFF') || false)
```

Comment:
Please update `org-information-interface.ts` with "orgType". Is it an array of strings?

Also, is orgType really optional? If not then please remove `?`.

Reply:
Oh sorry, I see you already added it to the interface!

Reply:
So it's a string of tokens?

Reply:
Updated to string comparision

Reply:
Oh, OK, that's clear to me now! Thanks.